src/img_web/app/forms.py

```diff
@@ -261,6 +261,11 @@ class ImageJobForm(forms.Form):
         required=False,
         widget=forms.HiddenInput(attrs={'readonly': 'readonly'}),
     )
+    devicebrand = forms.CharField(
```

Comment:
I think this field needs to be called just 'brand'.

The code that fills in the data-attributes for the template choices will add 'data-brand' attribute in there. And when the template is selected, the javascript on the page fills in the form fields based on the 'data-<fieldname>' attributes. So the field name needs to match the data- attribute, which gets the name ks attribute that is just 'Brand'

Reply:
fixed